tests/layer_tests/py_frontend_tests/test_torch_frontend.py

```diff
@@ -295,3 +295,26 @@ def add_ext(front_end, stat):
     assert tel_stat["send_event"] == 2
     assert tel_stat["send_error"] == 0
     assert tel_stat["send_stack_trace"] == 0
+
+
+def test_state_dict_names():
+    from openvino.frontend.pytorch.ts_decoder import TorchScriptPythonDecoder
+
+    import torchvision
+    model = torch.hub.load("pytorch/vision", "resnet18", weights="DEFAULT")
```

Comment:
please, add these commands https://github.com/openvinotoolkit/openvino/blob/master/.github/workflows/windows.yml#L117-L135 to Windows job running these tests